__tests__/CommentsTouchBox-test.js

```diff
@@ -0,0 +1,100 @@
+import 'react-native';
+import React from 'react';
+import CommentsTouchBox from '@screens/app/Components/CommentsTouchBox/CommentsTouchBox';
+import renderer from 'react-test-renderer';
+import {shallow} from 'enzyme';
+import checkPropTypes from 'check-prop-types';
+var assert = require('assert');
+
+test('like button snapshot test', () => {
+  const treeWithCommentLengthZero = renderer
+    .create(<CommentsTouchBox commentsLength={0} />)
+    .toJSON();
+  expect(treeWithCommentLengthZero).toMatchSnapshot();
+  const treeWithCommentLengthOne = renderer
+    .create(<CommentsTouchBox commentsLength={1} />)
+    .toJSON();
+  expect(treeWithCommentLengthOne).toMatchSnapshot();
+  const treeWithCommentLengthGreaterThanOne = renderer
+    .create(<CommentsTouchBox commentsLength={2} />)
+    .toJSON();
+  expect(treeWithCommentLengthGreaterThanOne).toMatchSnapshot();
+});
```

Comment:
What about snapshot with len < 0? or MAX_INT? Do we have a min/max set? If so, should we create snapshots for those edge cases